### PR TITLE
added get matches by player id

### PR DESCRIPTION
**Pull request by Mahooki**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tokenExpirySeconds=1200
```
Comment (Mahooki) [inline src/main/resources/application.properties:1]: Added these for local run. Not sure if this file would override environment variable configurations on AWS. We can remove this if needed.

Comment (Eirches) [reply src/main/resources/application.properties:1]: Short answer - no problem.

Long answer - check the documentation at: https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html

Comment (yadawg) [reply src/main/resources/application.properties]: Seems like we should remove them unless AWS pulls those from somewhere else. @Eirches should we remove these?

**Review by yadawg — APPROVED** [review]
Looks good, neat that there's external properties options for the `application.properties` variables we were wondering about

**Review by Eirches — APPROVED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (Mahooki) [reply src/main/resources/application.properties:1]: We decided to put these in a application-profile.properties file. This way it'll still ship with the app, but won't be applied unless specified on AWS. An alternative to this strategy would be to put it outside of the project so it doesn't get jarred up, but add it via classpath locally.

**Pull request merged**